Mathlib/AlgebraicGeometry/ProjectiveSpectrum/Scheme.lean

```diff
@@ -294,7 +294,12 @@ theorem mem_carrier_iff_of_mem (hm : 0 < m) (q : Spec.T A⁰_ f) (a : A) {n} (hn
   trans (HomogeneousLocalization.mk ⟨m * n, ⟨proj 𝒜 n a ^ m, by rw [← smul_eq_mul]; mem_tac⟩,
     ⟨f ^ n, by rw [mul_comm]; mem_tac⟩, ⟨_, rfl⟩⟩ : A⁰_ f) ∈ q.asIdeal
   · refine ⟨fun h ↦ h n, fun h i ↦ if hi : i = n then hi ▸ h else ?_⟩
-    convert zero_mem q.asIdeal
+    #adaptation_note
+    /--
+    After https://github.com/leanprover/lean4/pull/5376
+    we need to specify the implicit arguments by `inferInstance`.
+    -/
+    convert @zero_mem _ _ inferInstance inferInstance _ q.asIdeal
```

Comment:
Wouldn't this be fixed by making `M` an `outParam` in `ZeroMemClass S M`? Let me try...

Reply:
I have a branch where I did this. Let me push it. (Alternately you can use `Ideal.zero_mem`)

Reply:
https://github.com/leanprover-community/mathlib4/tree/mrb/5376

Reply:
I am still messing around with trying to remove the adaptation note for the Kummer extensions so changes around there might be spurious

Reply:
There should probably be a library note about requiring `outParam`s